keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -261,6 +265,35 @@ Unit, integration and E2E tests cover the existing StatefulSet mechanics.
 Additionally, unit and integration tests will be added to cover the
 API validation, behavioral change of StatefulSet with feature gate enabled and disabled.
 
+#### Prerequisite testing updates
+
+[x] I/we understand the owners of the involved components may require updates to existing tests to make this code solid enough prior to committing the changes necessary to implement this enhancement.
+
+##### Unit tests
+
+
+`k8s.io/kubernetes/pkg/apis/apps/validation`                       `06/07/2022`:    `90.6% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:93`:	   `06/07/2022`:		`93.1% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:162`:	   `06/07/2022`:		`100% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:169`:	   `06/07/2022`:		`94.1% of statements`
+`k8s.io/kubernetes/pkg/apis/apps/validation/validation.go:199`:	   `06/07/2022`:		`100% of statements`
+`k8s.io/kubernetes/pkg/controller/statefulset`:                    `06/07/2022`:    `85.5% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset`:                 `06/07/2022`:    `76.7% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:95`:	 `06/07/2022`:	  `100.0% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:139`: `06/07/2022`:	  `100.0% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:223`: `06/07/2022`:	  `100.0% of statements`
+`k8s.io/kubernetes/pkg/registry/apps/statefulset/strategy.go:118`: `06/07/2022`:	  `100.0% of statements`
+
+##### Integration tests
+
+Added integration tests to test availabile replicas when minReadySeconds is set on the statefulset spec.
```

Comment:
:+1: also from template:
```
For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
https://storage.googleapis.com/k8s-triage/index.html
```

Reply:
+1 - please link those

Reply:
Added a link for the test here: https://github.com/kubernetes/enhancements/pull/3354/files#diff-08898e79412d6aad472d55e637efe7f433a78688ac74d94b6d1283b357a1a0a3R292.

Can we have integration tests in the triage dashboard you linked. When I searched it shows only e2es unless I missed something.